examples/routing_service/routing_service_mongo_db/cpp/README.rst

```diff
@@ -93,14 +118,26 @@ To run Routing Service, you will need first to set up your environment as follow
 
 .. code::
 
-    $export RTI_LD_LIBRARY_PATH=<Connext DDS Directory>/lib/<Connext DDS Architecture>
+    export RTI_LD_LIBRARY_PATH=<Connext DDS Directory>/lib/<Connext DDS Architecture>
+
+.. note::
+
+    If you wish to run the executable for a target architecture different
+    than the host, you have to specify your architecture. That way the Routing
+    Service script can use the specific target binary instead of using the standard
+    host binary. This can be done by using the ``CONNEXTDDS_ARCH`` variable.
+
+    .. code::
+
+    export CONNEXTDDS_ARCH=<Connext DDS Architecture>
```

Comment:
Mention a real architecture to be consistent. ``export CONNEXTDDS_ARCH=x64Linux3gcc5.4.0``